Parser/Lexer/Tokenizer.cs

```diff
@@ -54,14 +52,14 @@ public class Tokenizer {
    // If we see an open " then we construct a string token
    Token String () {
       int start = mN++;
-      while (mN < mText.Length && mText[mN++] != '"') { }
+      while (mN < mText.Length && mText[mN++] != '"') ;

```

Comment:
This is not good style. It will be confusing since it looks like you added a ; by mistake. The { } style is preferred.